content/blog/federating-spire-on-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,441 @@
+```
+
+## 4﻿.2 Deploy Bookinfo Sample Application in Cluster-2
+
+In Cluster 2, apply a new ClusterSpiffeID called federated that registers resources with the label **spiffe.io/spire-managed-identity=spire** that can be federated with cluster1. Apply the bookinfo sample application manifest. 
+
+```shellsession
+kubectl apply -f services/istio/release-1.17/bookinfo.yaml 
+```
+
+## 4﻿.3 Check if all the resources created are up and running:
+
+```shellsession
+Cluster1:~ # kubectl get po 
+
+NAME                              READY   STATUS    RESTARTS   AGE 
+
+curl-greeter                      2/2     Running   0          15h 
+```
+
+```shellsession
+Cluster2:~ # kubectl get po 
+
+NAME                                READY   STATUS    RESTARTS   AGE 
+details-v1-bff8759df-vkvb4          2/2     Running   0          16h 
+greeter-client-76686757cd-6j2ft     2/2     Running   0          21h 
+productpage-v1-98887b9b-x5k24       2/2     Running   0          16h 
+ratings-v1-7ddbb859fc-htmfq         2/2     Running   0          16h 
+reviews-v1-67b576c8bf-jr6tj         2/2     Running   0          16h 
+reviews-v2-7ffbdcc5f7-m2c29         2/2     Running   0          16h 
+reviews-v3-6dbfcc6d89-zn9tw         2/2     Running   0          16h 
+```
+
+## 4﻿.4 Check that the workload identity was issued by SPIRE
+
+Spire Certificate at Curl-Greeter 
+
+```shellsession
+Cluster1:~ # istioctl proxy-config secret curl-greeter -o json | jq -r \ 
+
+> '.dynamicActiveSecrets[0].secret.tlsCertificate.certificateChain.inlineBytes' | base64 --decode > chain.pem 
+
+Cluster1:~ # openssl x509 -in chain.pem -text | grep SPIRE 
+
+        Subject: C = US, O = SPIRE, x500UniqueIdentifier = c3dd29b5f4a326f8a567854407456ea9 
+```
+
+*Note: Similarly, certificates for other pods can be checked.* 
+
+## 4﻿.5 Create a secret to hold bookinfo product page credentials like this: 
+
+```shellsession
+kubectl create -n istio-system secret generic bookinfo-credential \ 
+
+\--from-file=tls.key=bookinfo.com.key \ 
+
+\--from-file=tls.crt=bookinfo.com.crt \ 
+
+\--from-file=ca.crt=bookinfo.ca.crt
+```
+
+Obtain the tls cert and key using this command: 
+
+```shellsession
+kubectl exec -n spire -c spire-server deployment/spire-server -- /opt/spire/bin/spire-server x509 mint -spiffeID spiffe://cluster2.demo/ns/default/sa/bookinfo-productpage
+```
+
+*Note: Copy the SVID section into a new file bookinfo.com.crt and Private key section into bookinfo.com.key.* 
+
+Obtain the ca cert using this command: 
+
+```shellsession
+kubectl exec -n spire -c spire-server deployment/spire-server -- /opt/spire/bin/spire-server bundle list
+```
+
+*Note: Copy the ca cert under the cluster1.demo section into a new file bookinfo.ca.cert.* 
+
+Apply the gateway configuration for the bookinfo application found here: (It uses the istio-ingress gateway) 
```

Comment:
Are you missing a link? You say "found here". That usually indicates a hot-link. Perhaps you meant "for the bookinfo application shown below:" instead?